server/user/bitrot-detect.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=the bit-rot detection script
```

Comment:
Actually, those descriptions are emitted by systemd when you use `systemctl --user status bitrot-detect[.timer]`.  The were crafted so that output made sense.